docs/maintainers_guide.rst

```diff
@@ -32,14 +32,27 @@ change:
     a change already merged onto ``main``, unless the change simply does
     not make sense on ``main``.
 
+At any given time Qiskit has up to two supported stable branches, one for the
+previous minor version release series and the other for the final minor
+release series on the previous major release. For example, if the ``main``
+branch is developing the 1.2.0 release, then the ``stable/1.1`` branch and
+potentially the ``stable/0.46`` branches are supported.
 
 Backporting
 -----------
 
 When a PR tagged with ``stable backport potential`` is merged, or when a
 merged PR is given that tag, the `Mergify bot <https://mergify.com>`__ will
-open a PR to the current stable branch.  You can review and merge this PR
-like normal.
+open a PR to the most recent supported stable branch. You can review and merge
+this PR like normal. If you're backporting from main to a different stable
+branch you can leave a comment of the form ``@Mergifyio backport branch`` on a
```

Comment:
Would it be worth adding a `previous stable backport potential` label to .mergify.yml?

Reply:
Oh, yeah this is a good idea we can totally do this. I'll update the docs to include this.

Reply:
If we're going this route, I wouldn't mind doing it as explicit labels (`backport to 0.45`, `backport to 1.1`, etc) - it's more work to manage Mergify since we can't easily make it dynamic on the label, but I already sometimes forget what "stable backport potential" means when we're in an rc period.